src/storage/table_test.cc

```diff
@@ -504,26 +741,51 @@ TEST_F(TableTest, TableIteratorCount) {
         count++;
         it->Next();
     }
-    ASSERT_EQ(44471, count);
+    if (storageMode == ::openmldb::common::StorageMode::kMemory) {
+        ASSERT_EQ(44471, count);
+    } else {
+        ASSERT_EQ(55551, count);
```

Comment:
how to calculate these two numbers?

Reply:
We only have 55551 now.
We seek to pk500 9528. The next data is pk500 9527. Then all these data with pk greater than str(500) and both timestamps are OK. 
Which means, 
Next Data pk500 9527, 1
From pk6 to pk8, 4
From pk52 to pk98, 48
From pk502 to pk998, 498
From pk5000 to pk 9998, 5000
From pk50000 to pk99998, 50000
Total is 55551

Reply:
can add a comment here:
`We seek to pk500 9528. The next data is pk500 9527. Then all these records with pk greater than str(500) and both timestamps are traversed`